api/CampusService.proto

```diff
@@ -64,6 +64,51 @@ service Campus {
       response_body: "events"
     };
   }
+
+
+  rpc GetCafeteriaRatingLastThree (GetCafeteriaRating) returns (GetCafeteriaRatingReply) {
+    option (google.api.http) = {
+      post: "/cafeteriaRating/cafeteria/get:cafeteriaName?limit=3"
```

Comment:
Generally I'd recommend not to do casing in the APIs. This can be simplified to: `/cafeteria/rating/get` - Is there an alternative guideline you are following? What is the intended function of adding `?limit=3` here? (I'm guessing you are defining a default value?)

Reply:
I rather used ignorance, in describing the URL, as alternative guidlines. :) This was supposed to force a limit to be specified in the url, but it is removed now.